po/es.po

```diff
@@ -0,0 +1,1212 @@
+"La sesión incluye una breve plática, una demostración de programación en vivo y una "
+"sesión de preguntas al final."
+
+#: src\front-matter.md:11
+msgid ""
+"We conduct all training sessions remotely using modern video-conferencing "
+"tools to ensure the best learning experience."
+msgstr ""
+"Llevamos a cabo todas las sesiones de entrenamiento de manera remota con herramientas "
+"modernas de video-conferencias para asegurarnos de la mejor experiencia educativa."
+
+#: src\front-matter.md:14
+msgid "This repository contains the teaching material for this course."
+msgstr "Este repositorio contiene el material de enseñanza para este curso."
+
+#: src\front-matter.md:16
+msgid "Learning Goals"
+msgstr "Metas de aprendizaje"
+
+#: src\front-matter.md:18
+msgid ""
+"These are the questions you will be able to answer after attending this "
+"course:"
+msgstr ""
+"Estas son las preguntas que podrás responder después de acudir a este "
+"curso:"
+
+#: src\front-matter.md:21
+msgid "Is Ferrocene a fork of Rust?"
+msgstr "¿Es Ferrocene un fork de Rust?"
+
+#: src\front-matter.md:22
+msgid "What support is available for Ferrocene?"
+msgstr "¿Qué soporte hay disponible para Ferrocene?"
+
+#: src\front-matter.md:23
+msgid "What is it like using Ferrocene?"
+msgstr "¿Cómo es la experiencia de usar Ferrocene?"
+
+#: src\front-matter.md:25
+msgid "Timetable"
+msgstr "Horario"
+
+#: src\front-matter.md:27
+msgid "Our standard timetable for this course is as follows:"
+msgstr "Nuestro horario estándar es el siguiente:"
+
+#: src\front-matter.md:29
+msgid "Duration"
+msgstr "Duración"
+
+#: src\front-matter.md:29
+msgid "Contents"
+msgstr "Contenido"
+
+#: src\front-matter.md:31
+msgid "0:10"
+msgstr ""
+
+#: src\front-matter.md:31
+msgid "Room open, meet and greet"
+msgstr "Sesión de apertura, meet and greet"
+
+#: src\front-matter.md:32 src\front-matter.md:34
+msgid "0:30"
+msgstr ""
+
+#: src\front-matter.md:32
+msgid "Session 1 - What is Ferrocene?"
+msgstr "Sesión 1 - ¿Qué es Ferrocene?"
+
+#: src\front-matter.md:33
+msgid "0:05"
+msgstr ""
+
+#: src\front-matter.md:33
+msgid "Break"
+msgstr "Descanso"
+
+#: src\front-matter.md:34
+msgid "Session 2 - A Live Demo"
+msgstr "Sesión 2 - Live Demo"
+
+#: src\front-matter.md:35
+msgid "0:15"
+msgstr ""
+
+#: src\front-matter.md:35
+msgid "Q&A and Close"
+msgstr "Preguntas y Respuestas y Conclusión"
+
+#: src\just-rust.md:3
+msgid "No, really..."
+msgstr "No, en serio..."
+
+#: src\just-rust.md:5
+msgid "We didn't write a new Rust toolchain"
+msgstr "No escribimos un nuevo toolchain de Rust"
+
+#: src\just-rust.md:6
+msgid "We qualified The Rust Toolchain"
+msgstr "Cualificamos El Toolchain de Rust"
+
+#: src\just-rust.md:8
+msgid "Non-divergence"
+msgstr "No-divergencia"
+
+#: src\just-rust.md:10
+msgid ""
+"One of the Ferrocene pillars is that the standard library and the compiler "
+"must not diverge from upstream."
+msgstr ""
+"Uno de los pilares de Ferrocene es que la biblioteca estándar y el compilador "
```

Comment:
Here we say "libreria" instead of "biblioteca", which is pretty ironic considering that the first one is a fake friend.